test_regress/t/t_gantt_io.out

```diff
@@ -4,33 +4,44 @@ Argument settings:
   +verilator+prof+exec+start+2
   +verilator+prof+exec+window+2
 
-Analysis:
-  Total threads             = 2
-  Total mtasks              = 7
-  Total cpus used           = 2
-  Total yields              = 0
-  Total evals               = 2
-  Total eval loops          = 2
-  Total eval time           = 21875 rdtsc ticks
-  Longest mtask time        = 1190 rdtsc ticks
-  All-thread mtask time     = 5495 rdtsc ticks
-  Longest-thread efficiency = 5.4%
-  All-thread efficiency     = 12.6%
-  All-thread speedup        = 0.3
-
-Prediction (what Verilator used for scheduling):
-  All-thread efficiency     = 63.2%
-  All-thread speedup        = 1.3
+Summary:
+  Total elapsed time = 23415 rdtsc ticks
+  Parallelized code  = 82.51% of elapsed time
+  Total threads      = 2
+  Total CPUs used    = 2
+  Total mtasks       = 7
+  Total yields       = 0
+
+Parallelized code, measured:
+  Thread utilization =  14.22%
+  Speedup            =  0.284x
+
+Parallelized code, predicted during static scheduling:
+  Thread utilization =  63.22%
+  Speedup            =   1.26x
+
+All code, measured:
+  Thread utilization =  20.48%
+  Speedup            =   0.41x
+
+All code, measured, scaled by predicted speedup:
+  Thread utilization =  56.80%
+  Speedup            =   1.14x
 
 MTask statistics:
+  Longest mtask id = 5
+  Longest mtask time = 6.16% of time elapsed in parallelized code
   min log(p2e) = -3.681  from mtask 5 (predict 30, elapsed 1190)
   max log(p2e) = -2.409  from mtask 8 (predict 107, elapsed 1190)
   mean = -2.992
   stddev = 0.459
   e ^ stddev = 1.583
 
-CPUs:
-  cpu 10: cpu_time=4725 socket=0 core=10  Test Ryzen 9 3950X 16-Core Processor
-  cpu 19: cpu_time=770 socket=0 core=3  Test Ryzen 9 3950X 16-Core Processor
+CPU info:
+   Id | MTask   | Socket | Core | Model
+      | time    |        |      |
+  ====|=========|========|======|======
+   10 |  20.18% |      0 |   10 | Test Ryzen 9 3950X 16-Core Processor
+   19 |   3.29% |      0 |    3 | Test Ryzen 9 3950X 16-Core Processor
```

Comment:
Can you show the raw cpu time too please?

Reply:
Does this work for you?
```
CPU info:
   Id | Time spent executing MTask | Socket | Core | Model
      | % of elapsed ticks / ticks |        |      |
  ====|============================|========|======|======
    4 |   1.80% /              278 |      0 |    4 | 11th Gen Intel(R) Core(TM) i9-11950H @ 2.60GHz
   14 |  27.74% /             4288 |      0 |    6 | 11th Gen Intel(R) Core(TM) i9-11950H @ 2.60GHz
```

Reply:
Yes thanks